src/routes/email/entities/delete-email-dto.entity.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class DeleteEmailDto {
```

Comment:
We are missing the `timestamp` and `signature` keys.

Reply:
Added in 27eaae4e907918ecba4248fe5b52aceb7c32a10e.